src/coreComponents/common/GeosxConfig.hpp.in

```diff
@@ -164,6 +155,15 @@
 /// Version information for suitesparse
 #cmakedefine suitesparse_VERSION @suitesparse_VERSION@
 
+/// Version information for hypre
+#cmakedefine hypre_VERSION @hypre_VERSION@
```

Comment:
does this give equivalent information as was done by the removed code in `HypreInterface.cpp`

Reply:
Yes, it's taken from the same hypre macros, the only difference is it's now extracted at config-time rather than compile-time.